redash/models.py

```diff
@@ -436,6 +437,9 @@ def is_api_user(self):
 
     @property
     def gravatar_url(self):
```

Comment:
Let's rename this to `profile_image_url` (here and everywhere else we use it) to be consistent.